### PR TITLE
New Quickstart: "Getting Started with DevOps in Snowflake"

### PR DESCRIPTION
**Pull request by sfc-gh-vraudszus**
Quickstart focusing on DevOps features like SnowflakeCLI, Git integration, CREATE OR ALTER and Jinja Templating. The features are demonstrated on the example of a data pipeline combining marketplace data to find the perfect vacation spot.

### CODE REVIEW

File: site/sfguides/src/getting_started_with_devops_in_snowflake/getting_started_with_devops_in_snowflake.md [hunk 1]
```diff
@@ -0,0 +1,586 @@
...
+- SnowflakeCLI tool
+- Data Sharing/Marketplace (instead of ETL)
+- LLM interference with Snowflake Cortex
+
+### What you will need
+
+You will need the following things before starting:
+
+- Snowflake
+  - **A Snowflake Account**
```
Comment (iamontheinet) [inline site/sfguides/src/getting_started_with_devops_in_snowflake/getting_started_with_devops_in_snowflake.md]: @sfc-gh-vraudszus Add a link to signup/login.

File: site/sfguides/src/getting_started_with_devops_in_snowflake/getting_started_with_devops_in_snowflake.md [hunk 2]
```diff
@@ -0,0 +1,586 @@
...
+- Create a vectorized Python UDF making use of dynamic file access
+- Create a data engineering pipeline to process data from multiple sources
+- Orchestrate the pipeline with tasks
+- Make declarative changes to the pipeline with Create or Alter
+- Separate dev and prod environments with Jinja templating
+- Deploy the pipeline via an automated CI/CD pipeline
+- Enrich your data with Cortex LLM functions 
+
+<!-- ------------------------ -->
+## Quickstart Setup
```
Comment (iamontheinet) [inline site/sfguides/src/getting_started_with_devops_in_snowflake/getting_started_with_devops_in_snowflake.md]: @sfc-gh-vraudszus Rename this to Setup Environment

File: site/sfguides/src/getting_started_with_devops_in_snowflake/getting_started_with_devops_in_snowflake.md [hunk 3]
```diff
@@ -0,0 +1,586 @@
...
+
+There is a problem preventing us from doing so: Our pipeline definitions contain hard references to the production database “quickstart_prod”. We need to parametrize these references to enable a CI/CD pipeline to choose the deployment target.
+
+### Jinja Templating in Snowflake
+
+[Jinja](https://jinja.palletsprojects.com/en/3.1.x/) is a popular templating language with features ranging from simple script parametrization to extensive scripting in a language similar to Python. Snowflake supports the execution of templated SQL scripts with the [EXECUTE IMMEDIATE FROM](https://docs.snowflake.com/en/sql-reference/sql/execute-immediate-from) command.
+
+For the use case of changing the deployment target, variable replacement is all we need from Jinja. Jinja variables in SQL scripts are marked by `{{...}}`. We can provide a value for a variable in the USING clause of EXECUTE IMMEDIATE FROM. Template variables can be used everywhere is a SQL script. We can use variables to inject object names, property values or even entire statements.
+
+### Parametrize the Deployment Target
```
Comment (iamontheinet) [inline site/sfguides/src/getting_started_with_devops_in_snowflake/getting_started_with_devops_in_snowflake.md]: @sfc-gh-vraudszus Correct typo "Parameterize"

File: site/sfguides/src/getting_started_with_devops_in_snowflake/getting_started_with_devops_in_snowflake.md [hunk 4]
```diff
@@ -0,0 +1,586 @@
...
+
+<!-- ------------------------ -->
+## Teardown
+
+Duration: 1
+
+Once you're finished with the Quickstart and want to clean things up, you can drop all the objects you created by running the teardown script. Open `steps/08_teardown.sql` in the VS Code extension and run all command using the "Execute All Statements" button in the upper right corner of the editor window.
+
+<!-- ------------------------ -->
+## Conclusion
```
Comment (iamontheinet) [inline site/sfguides/src/getting_started_with_devops_in_snowflake/getting_started_with_devops_in_snowflake.md]: @sfc-gh-vraudszus Rename it to Conclusion And Resources

File: site/sfguides/src/getting_started_with_devops_in_snowflake/getting_started_with_devops_in_snowflake.md [hunk 5]
```diff
@@ -0,0 +1,586 @@
...
+
+- [Source Code on GitHub](https://github.com/Snowflake-Labs/sfguide-getting-started-with-devops-in-snowflake)
+- [DevOps in Snowflake: How Git and Database Change Management enable a file-based object lifecycle](https://medium.com/snowflake/devops-in-snowflake-how-git-and-database-change-management-enable-a-file-based-object-lifecycle-1f61a0d5257c)
+- [DevOps: Database Change Management with Terraform and GitHub](https://quickstarts.snowflake.com/guide/devops_dcm_terraform_github/#0)
+- [Advanced: Data Engineering Pipelines with Snowpark Python](https://quickstarts.snowflake.com/guide/data_engineering_pipelines_with_snowpark_python/#0)
+- Related Tools
+  - [Snowflake Visual Studio Code Extension](https://docs.snowflake.com/en/user-guide/vscode-ext)
+  - [SnowflakeCLI Tool](https://docs.snowflake.com/en/developer-guide/snowflake-cli-v2/index) and [GitHub Action](https://github.com/Snowflake-Labs/snowflake-cli-action)
+
+### Feedback
```
Comment (iamontheinet) [inline site/sfguides/src/getting_started_with_devops_in_snowflake/getting_started_with_devops_in_snowflake.md]: @sfc-gh-vraudszus Delete this section

**Review by iamontheinet — CHANGES_REQUESTED** [review]

File: site/sfguides/src/getting_started_with_devops_in_snowflake/getting_started_with_devops_in_snowflake.md [hunk 6]
```diff
@@ -0,0 +1,582 @@
...
+Are you interested in bringing your database game to the next level by unleashing the power of Snowflake’s many DevOps capabilities, empowering you to build safe and robust data pipelines? Well then, this Quickstart is for you!
+
+This quickstart will cover a lot of ground. By the end of this guide, you will have built a robust CI/CD pipeline that automates the deployment of changes to a data pipeline from Git, promotes the separation of dev and prod databases and facilitates monitoring. You'll also learn how to use Snowflake's marketplace to enrich your data without incurring storage costs, get to know the new developer CLI tool and the Visual Studio Code extension!
+
+![DevOps Overview](assets/devops-overview.png)
+
+The data pipeline used as an example in this quickstart makes use of data available on the Snowflake marketplace to come up with the perfect spot for a family vacation. Let's get started!
+
+> aside negative
+> Note: The Git integration, Create or Alter and Jinja templating are still in preview.
```
Comment (sfc-gh-twollnik) [inline site/sfguides/src/getting_started_with_devops_in_snowflake/getting_started_with_devops_in_snowflake.md]: wording sounds off. Do we have standard wording for preview disclaimers?

File: site/sfguides/src/getting_started_with_devops_in_snowflake/getting_started_with_devops_in_snowflake.md [hunk 7]
```diff
@@ -0,0 +1,582 @@
...
+- Fundamentals of data engineering. You can use [Intro to Data Engineering with Snowpark Python](https://quickstarts.snowflake.com/guide/data_engineering_with_snowpark_python_intro/index.html) to refresh your memory
+- Familiarity with Git repositories and GitHub
+
+### What you will learn
+
+You will learn about the following Snowflake features in this Quickstart:
+
+- Native Git Integration
+- Declarative Database Change Management
+- Parametrization of SQL scripts with Jinja
```
Comment (sfc-gh-twollnik) [inline site/sfguides/src/getting_started_with_devops_in_snowflake/getting_started_with_devops_in_snowflake.md]: include the keyword "templating"?

File: site/sfguides/src/getting_started_with_devops_in_snowflake/getting_started_with_devops_in_snowflake.md [hunk 8]
```diff
@@ -0,0 +1,582 @@
...
+### What you will build
+
+During this Quickstart you will accomplish the following things:
+
+- Establish a connection between Snowflake and a GitHub repository
+- Setup access to Snowflake Marketplace data
+- Create a vectorized Python UDF making use of dynamic file access
+- Create a data engineering pipeline to process data from multiple sources
+- Orchestrate the pipeline with tasks
+- Make declarative changes to the pipeline with Create or Alter
```
Comment (sfc-gh-twollnik) [inline site/sfguides/src/getting_started_with_devops_in_snowflake/getting_started_with_devops_in_snowflake.md]: wondering if we should make it more clear that it is not "either create or alternatively alter", but rather a combined new feature. Suggestion: Create-or-Alter. If you change this there are many places in the doc that need to be updated consistently.

**Review by sfc-gh-twollnik — COMMENTED** [review]
didn't yet have time to go over everything, but first impression is very positive!

File: site/sfguides/src/getting_started_with_devops_in_snowflake/getting_started_with_devops_in_snowflake.md (hunk 6 above)
Comment (sfc-gh-vraudszus) [reply site/sfguides/src/getting_started_with_devops_in_snowflake/getting_started_with_devops_in_snowflake.md]: @iamontheinet

**Pull request merged**